### PR TITLE
Feature/delete credential

### LINKED ISSUE #472: Create new command rit delete credential
**What would you like to add**:
Command to delete a specific credential.

**Why this is necessary**:
To facilitate credential management

### PR DESCRIPTION
**Pull request by matheussn**
### Description

I've implemented a functionality for the command `rit delete credential` which show the credentials created in the current context to deleting.
closes #472 

### How to verify it
To validate, you must create a credential in the current context, using the `rit set credential` command. Once created, you will be able to use the `rit delete credential` command to delete it

### Changelog

Show the option to delete your credentials on rit delete credential command.

### CODE REVIEW

**Review by henriquemoraeszup — COMMENTED** [review]
Nice tests 🚀

File: pkg/cmd/delete_credential.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+func (d deleteCredentialCmd) runStdin() CommandRunnerFunc {
+	return func(cmd *cobra.Command, args []string) error {
+		dc, err := d.stdinResolver(cmd.InOrStdin())
+		if err != nil {
+			return err
+		}
+
+		var mustDelete = false
```
Comment (victorschumacherzup) [inline pkg/cmd/delete_credential.go:107]: i think a declaration without `var` would match better with the other variables

**Review by victorschumacherzup — APPROVED** [review]

File: pkg/cmd/delete.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ func NewDeleteCmd() *cobra.Command {
 		Short:     "Delete contexts, repositories, formulas and workspaces",
 		Long:      "Delete contexts, repositories, formulas and workspaces",
```
Comment (brunasilvazup) [inline pkg/cmd/delete.go:26]: Did you can add "credential" in the description too?

**Comment by henriquemoraeszup** [comment]
Hey matheus! I am good to approve but the settings will dismiss my review if you correct the conflicts, just let me know when you are done =]

**Review by brunasilvazup — COMMENTED** [review]
Do you can add the new command to `CoreCmds` in `api.go`

**Comment by matheussn** [comment]
> Hey matheus! I am good to approve but the settings will dismiss my review if you correct the conflicts, just let me know when you are done =]

Hello @henriquemoraeszup 
 I believe everything is ready!

File: pkg/credential/delete_test.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+				},
+				fm: fileRemoverErrorMock{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := NewCredDelete(tt.fields.homePath, tt.fields.cf, tt.fields.fm)
+			if err := got.Delete(tt.fields.service); (err != nil) != tt.wantErr {
+				t.Errorf("Find(%s) got %v, wantErr %v", tt.name, err, tt.wantErr)
```
Comment (brunasilvazup) [inline pkg/credential/delete_test.go:84]: Typo `Find(...` -> `Delete(...` ?

File: pkg/cmd/delete_credential.go [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+	credential.CredDelete
+	credential.ReaderPather
+	rcontext.Finder
+	prompt.InputBool
+	prompt.InputList
+}
+
+// deleteCredential type for stdin json decoder
+type deleteCredential struct {
+	Service string `json:"service"`
```
Comment (brunasilvazup) [inline pkg/cmd/delete_credential.go:25]: I think "provider" would be more contextualized, what do you think?

File: pkg/cmd/delete_credential.go [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+		ValidArgs: []string{""},
+		Args:      cobra.OnlyValidArgs,
+	}
+	cmd.LocalFlags()
+	return cmd
+}
+
+func (d deleteCredentialCmd) runPrompt() CommandRunnerFunc {
+	return func(cmd *cobra.Command, args []string) error {
+		context, err := d.getCurrentContext()
```
Comment (brunasilvazup) [inline pkg/cmd/delete_credential.go:58]: To improve the experience, could you put an information sheet from the current context?
Something like: `fmt.Println("Current context: ", context)`

**Review by brunasilvazup — APPROVED** [review]

**Review by henriquemoraeszup — APPROVED** [review]

**Pull request merged**